docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -94,41 +115,56 @@ oc logs deployment.apps/operator-hazelcast-platform-operator
 
 == Step 2. Start the Hazelcast Cluster
 
-After installing and running the Hazelcast Platform Operator, you can create a Hazelcast cluster. First, create the `Hazelcast` custom resource file as `hazelcast.yaml`.
+After installing and starting Hazelcast Operator, you can create and start a Hazelcast cluster as follows:
 
 [tabs]
 ====
-Open Source::
+Enterprise Edition::
 +
 --
+Hazelcast Platform (Enterprise Edition) requires a license key. 
 
-. Create the `Hazelcast` custom resource file and name it `hazelcast.yaml`.
+If you don't have a license key, you can request a trial license from the link:http://trialrequest.hazelcast.com/[Hazelcast website].
+
+. Create a Kubernetes secret to hold your license key as follows:
 +
-[source,yaml,subs="attributes+"]
+.For Kubernetes
+[source,shell]
 ----
-include::ROOT:example$/hazelcast.yaml[]
+kubectl create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
+----
++
+.For Openshift
+[source,shell]
+----
+oc create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
 ----
 
-. Apply the custom resource to start the Hazelcast cluster.
+. Create the Hazelcast custom resource file with the following content, and save it in your repository as _hazelcast-enterprise.yaml_:
++
+TIP: This is the Hazelcast custom resource referred to throughout this documentation.
 +
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hazelcast-enterprise.yaml[]
+----
 
+. Apply the custom resource file to start the Hazelcast cluster as follows:
++
 .For Kubernetes
 [source,shell]
 ----
-kubectl apply -f hazelcast.yaml
-
+kubectl apply -f hazelcast-enterprise.yaml
```

Comment:
I needed to use the full path to the YAML as I am not in that directory (I'm in my helm directory, just for info)
Is this assumed knowledge? I worked it out, but we don't want people to have to work things out....

Reply:
It is assumed knowledge, users create their own YAML files so they should know their path.

Reply:
worth noting that if you do not run the command from the path containing your YAML file, you must provide the full path in the command?

Belt and braces for anyone who doesn't know - or is too busy following instructions to think about it!

Reply:
Decided not to add anything; we can do so later if it is felt to be useful in any way.